pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/guide.en-us.md

```diff
@@ -0,0 +1,1272 @@
+
+## Step 2 - Creating a TrilioVault Target to Store Backups
+
+`TrilioVault` needs to know first where to store your backups. TrilioVault refers to the storage backend by using the `target` term, and it's managed via a special `CRD` named `Target`. The following target types are supported: `S3` and `NFS`. For `OVH Cloud` and the purpose of the `Tuturial`, it makes sense to rely on the `S3` storage type because it's `cheap` and `scalable`. To benefit from an enhanced level of protection you can create multiple target types (for both `S3` and `NFS`), so that your data is kept safe in multiple places, thus achieving backup redundancy.
+
+Typical `Target` definition looks like below:
+
+```yaml
+apiVersion: triliovault.trilio.io/v1
+kind: Target
+metadata:
+  name: trilio-ovh-s3-target
+  namespace: tvk
+spec:
+  type: ObjectStore
+  vendor: Other								# e.g. AWS for AWS S3 Storage
+  enableBrowsing: true
+  objectStoreCredentials:
+    bucketName: <YOUR_OVH_OBJECT_STORAGE_BUCKET_NAME_HERE>
+    region: <YOUR_OVH_OBJECT_STORAGE_BUCKET_REGION_HERE>    		# e.g.: bhs5 for OVH Object Storage or 'us-est-1 etc' for AWS S3
+    url: "https://<YOUR_OVH_OBJECT_STORAGE_BUCKET_ENDPOINT_HERE>"  	# e.g.: storage.bhs5.cloud.ovh.net for OVH
+    credentialSecret:
+      name: trilio-ovh-s3-target
+      namespace: tvk
+  thresholdCapacity: 10Gi
+```
+
+Explanation for the above configuration:
+
+- `spec.type`: Type of target for backup storage (S3 is an object store).
+- `spec.vendor`: Third party storage vendor hosting the target (for `OVH Object Storage` you need to use `Other` instead of `AWS`).
+- `spec.enableBrowsing`: Enable browsing for the target to brose through the backups stored on it.
+- `spec.objectStoreCredentials`: Defines required `credentials` (via `credentialSecret`) to access the `S3` storage, as well as other parameters such as bucket region and name.
+- `spec.thresholdCapacity`: Maximum threshold capacity to store backup data.
+
+To access `S3` storage, each target needs to know bucket credentials. A `Kubernetes Secret` must be created as well:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: trilio-ovh-s3-target
+  namespace: tvk
+type: Opaque
+stringData:
+  accessKey: <YOUR_OVH_OBJECT_STORAGE_BUCKET_ACCESS_KEY_ID_HERE>	# value must be base64 encoded
+  secretKey: <YOUR_OVH_OBJECT_STORAGE_BUCKET_SECRET_KEY_HERE>    	# value must be base64 encoded
+```
+
+Notice that the secret name is `trilio-ovh-s3-target`, and it's referenced by the `spec.objectStoreCredentials.credentialSecret` field of the `Target` CRD explained earlier. The `secret` can be in the same `namespace` where `TrilioVault` was installed (defaults to `tvk`), or in another namespace of your choice. Just make sure that you reference the namespace correctly. On the other hand, please make sure to `protect` the `namespace` where you store `TrilioVault` secrets via `RBAC`, for `security` reasons.
```

Comment:
replace
`trilio-ovh-s3-target`
with
`trilio-ovh-s3-target-secret`

Reply:
Updated the secret name at all places. Thank you!